### ISSUE TITLE
NXDL Source link is wrong for contributed definitions

### ISSUE THREAD

**Issue by woutdenolf** [nexusformat/definitions#1144]
For example:

https://nexusformat.github.io/definitions/classes/contributed_definitions/NXaberration.html#nxaberration

It links to

https://github.com/nexusformat/definitions/blob/main/base_classes/NXaberration.nxdl.xml

but is should be

https://github.com/nexusformat/definitions/blob/main/contributed_definitions/NXaberration.nxdl.xml

**Comment by prjemian** [comment]
Is #1149 a duplicate of this issue? If so, @PeterC-DLS was testing his branch.  We're just waiting on that branch and a PR to review.

**Comment by PeterC-DLS** [comment]
With #1153, there will be fewer differences.
